Source/UI/NewUI/PaymentForm/View/SecurityCodeView.swift

```diff
@@ -19,7 +19,7 @@ public final class SecurityCodeView: UIView {
   private(set) var supportedScheme: Card.Scheme = .unknown
   private(set) var style: CellTextFieldStyle?
 
-  private lazy var codeInputView: InputView = {
+  private(set) lazy var codeInputView: InputView = {
```

Comment:
Were you planning to add some unit test for this? Otherwise it should be private